draft-salowey-wimse-arch.md

```diff
@@ -63,7 +63,25 @@ A workload is a running instance of software executing for a specific purpose th
 
 # Use Cases
 
-1. Basic Service Authentication
+## Initial Workload Identity
+
+Typically a workload obtains its identity early in its lifecycle. This identity is sometimes referred to as the "bottom turtle" on which further identity is built. Some common mechanisms for obtaining this initial identity include:
+
+* File System projection - in this mechanisms the identity is provisioned to the workload as an entity in the filesystem.
+* Local API - the identity is provided through an api such as a local domain socket (SPIFFE) or local Network API call (Cloud Provider Metadata Server)
+* Environment Injection - identity may also be injected into the workloads execution environment.
+
+### Attestation
+
+### Identity Credentials
+
+The identity is provisioned to the workload as a set of credentials. There are two main types of workload credentials: bearer tokens and X.509 certificates.
+
+Bearer tokens are tokens presented to another party as proof of identity.  They are typically signed to prevent forgery, however since these credentials are not bound to other information its possible that they could be stolen and reused elsewhere.
+
```

Comment:
I would add "To minimize risks of replay attacks, bearer tokens have short lifespan and a rotated often"

Otherwise it reads as if Bearer tokens is a non-starter.